code/modules/cards/cardhand.dm

```diff
@@ -6,26 +6,9 @@
 	w_class = WEIGHT_CLASS_TINY
 	worn_icon_state = "card"
 
-/obj/item/toy/cards/cardhand/Initialize(mapload, list/cards_to_combine = list())
+/obj/item/toy/cards/cardhand/Initialize(mapload)
 	. = ..()
 
-	var/has_runtime_spawned_cards = length(cards_to_combine)
-	var/has_mapped_spawned_cards = mapload && length(cards)
-
-	if(!has_runtime_spawned_cards && !has_mapped_spawned_cards)
-		CRASH("[src] is being made into a cardhand without a list of cards to combine")
-
-	if(has_mapped_spawned_cards) // these cards have not been initialized
-		for(var/card_name in cards)
-			var/obj/item/toy/singlecard/new_card = new (loc, card_name)
-			new_card.update_appearance()
-			cards_to_combine += new_card
-		cards = list() // reset our cards to an empty list
-
-	for(var/obj/item/toy/singlecard/new_card in cards_to_combine)
-		new_card.forceMove(src)
-		cards += new_card
-
```

Comment:
Unused

Reply:
https://github.com/tgstation/tgstation/blob/f4af90780aa1f8e6855945bde76bb82a862d7815/code/modules/cards/singlecard.dm#L171

Reply:
Ugh

Reply:
What if I just make that:
```
		var/obj/item/toy/cards/cardhand/new_cardhand = new (drop_location())
		new_cardhand.insert(card)
```

Reply:
Ok it doesn't work but I'll figure it out